Shop.API/Controllers/OrderController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Shop.BLL.Common.DataTransferObjects.Orders;
+using Shop.BLL.Interfaces;
+
+namespace Shop.API.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class OrderController(IOrderService orderService) : ControllerBase
+{
+    [HttpGet]
+    public async Task<IEnumerable<OrderResponseDto>> GetOrdersAsync()
+    {
+        return await orderService.GetOrdersAsync();
+    }
+
+    [HttpGet("{id:guid}")]
+    public async Task<OrderResponseDto> GetOrderById(Guid id)
```

Comment:
it is necessary to transfer the token to the controller method and then throw it into the service